app/javascript/src/ActiveDocs/customize/wrappedComponents/Parameters.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import 'ActiveDocs/customize/customizeOAS.scss'
+
+const findParamInService = (paramName, service) => {
+  return service[paramName]
+}
+
+const Select = ({value, className}) => {
+  return (
+    <div className={`autocomplete-select ${className}`}>
+      <select>
+        <option value={value}>{value}</option>
+      </select>
+    </div>
+  )
+}
+
+export const ParameterRow = (Original, system) => props => {
+  const paramList = props.rawParam._list._tail.array
+  const paramName = paramList.filter(param => param[0] === 'name')[0][1]
+
+  const customParam = findParamInService(paramName, system.service)
+  if (!customParam) {
+    return React.createElement(Original, props)
```

Comment:
Maybe this conditional doesn't belong here. You could craft this assuming you'll return the enhanced component and do the check before you instantiate it